client/src/components/layout/PlanetShow.js

```diff
@@ -8,7 +8,7 @@ import translateServerErrors from "../../services/translateServerErrors"
 const PlanetShow = (props) => {
   const [planet, setPlanet] = useState({
     name: "",
-    reviews: [],
+    reviews: [{ voteCount: []}]
```

Comment:
voteCount wont end up being an array, so this default state doesnt work this was from our conversation earlier as pseudocode!